packages/contracts-bedrock/src/universal/CrossDomainMessenger.sol

```diff
@@ -138,10 +135,14 @@ abstract contract CrossDomainMessenger is
     ///         successfully executed on the first attempt.
     mapping(bytes32 => bool) public failedMessages;
 
+    /// @notice Address of the paired CrossDomainMessenger contract on the other chain.
+    /// @custom:network-specific
+    address public otherMessenger;
```

Comment:
Can you make this be a `CrossDomainMessenger` type? It would be more consistent with other places in the codebase